docs/developers/developing.rst

```diff
@@ -64,9 +67,47 @@ extension itself, but not to h.
    `Extensions Reloader <https://chrome.google.com/webstore/detail/extensions-reloader/fimgfedafeadlieiabdeeaodndnlbhid?hl=en>`_
    to make this easier.
 
+.. _running-from-hosted-h:
+
+B. Running the Client Locally, Connected to Hosted h
+---------------------------------------------
+
+Method two is also very easy and only requires a little extra configuration in
+the client environment to setup. You also do not need to clone the browser extension
+repo. Assuming you have cloned the client repository, set the environment variable accordingly
+and run the local express server as follows:
+
+.. code-block:: sh
+
+   export SIDEBAR_APP_URL=https://hypothes.is/app.html
```

Comment:
I may be way off here, but won't this use the CDN embedded client instead of the local?